src/main/Dashboard.java

```diff
@@ -1,10 +1,131 @@
 package main;
 
+import java.util.HashSet;
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
 public class Dashboard {
 
-	public static void main(String[] args) {
-		// TODO Auto-generated method stub
+    static HashSet<User> hashUsers = new HashSet<User>();
+    static Scanner scanner = new Scanner(System.in);
+
+    public static void main(String[] args) {
+        System.out.println("Welcome to User Management System ");
+        int menu = 0;
+        String email;
+
+        do {
+            System.out.println("[0] EXIT");
+            System.out.println("[1] REGISTER USER");
+            System.out.println("[2] LIST ALL USERS");
+            System.out.println("[3] MODIFY USER");
+            System.out.println("[4] DELETE USER");
+
+            try {
+                System.out.println("Choose a menu: ");
+
+                while (!scanner.hasNextInt()) {
+                    scanner.next();
+                }
+
+                menu = scanner.nextInt();
+                scanner.nextLine();
+
+                switch (menu) {
+                case 0:
+                    // Quit
+                    System.out.println("GoodBye...");
+                    System.exit(0);
+                    break;
+
+                case 1:
+                    // Register User
+                    registerUser();
+                    break;
+
+                case 2:
+                    // List out all Users
+                    listUsers();
+                    break;
+
+                case 3:
+                    // Modify a User
+                    System.out.println("Please enter the current email of the user you wish to modify:");
+                    email = scanner.next().toString();
```

Comment:
No need to call `.toString()` on the return value of `scanner.next()`. It's already a `String` object.